app/kumactl/cmd/delete/delete_meshes.go

```diff
@@ -0,0 +1,41 @@
+package delete
+
+import (
+	"context"
+	"github.com/Kong/kuma/pkg/core/resources/apis/mesh"
+	"github.com/Kong/kuma/pkg/core/resources/model"
+	"github.com/Kong/kuma/pkg/core/resources/store"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func newDeleteMeshCmd(pctx *deleteContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "mesh",
+		Short: "Delete Mesh",
+		Long:  `Delete Mesh.`,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			rs, err := pctx.CurrentResourceStore()
+			if err != nil {
+				return err
+			}
+			meshName := args[0]
+
+			getOptions := store.GetByKey(model.DefaultNamespace, meshName, meshName)
+			meshDetails := mesh.MeshResource{}
+			if err := rs.Get(context.Background(), &meshDetails, getOptions); err != nil {
```

Comment:
It's not necessary to fetch a resource before deleting it

Reply:
I did a `get` to check if the mesh exists under the default namespace before deleting. 

[rs.Delete](https://github.com/Kong/kuma/blob/master/pkg/plugins/resources/memory/store.go#L149) is not throwing an error in case the mesh is not found.

Reply:
Ah, I see now.

It must be related to #190.

Ok, let's keep this `Get -> Delete` workflow then (and rework it once #190 is done)